base_layer/core/src/blocks/block_header.rs

```diff
@@ -412,12 +412,12 @@ impl ConsensusEncoding for BlockHeader {
         self.height.consensus_encode(writer)?;
         copy_into_fixed_array_lossy::<_, 32>(&self.prev_hash).consensus_encode(writer)?;
         self.timestamp.consensus_encode(writer)?;
-        self.output_mr.as_slice().consensus_encode(writer)?;
-        self.witness_mr.as_slice().consensus_encode(writer)?;
+        self.output_mr.as_fixed_slice().consensus_encode(writer)?;
```

Comment:
`FixedHash` already has a consensus encoding implementation
```suggestion
        self.output_mr.consensus_encode(writer)?;
```